rules/windows/execution_from_unusual_path_cmdline.toml

```diff
@@ -0,0 +1,40 @@
+[metadata]
+creation_date = "2020/10/30"
+maturity = "production"
+updated_date = "2020/10/30"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies process execution from suspicious default Windows directories. This is sometimes done by adversaries to hide
+malware in trusted paths.
```

Comment:
```suggestion
Identifies process execution from suspicious default Windows directories. This may be abused by adversaries to hide
malware in trusted paths.
```

Reply:
may need a re-lint